Xamarin.Forms.Platform.Android/Renderers/FileImageSourceHandler.cs

```diff
@@ -44,20 +44,39 @@ public Task LoadImageAsync(ImageSource imagesource, ImageView imageView, Cancell
 				if (uri != null)
 					imageView.SetImageURI(uri);
 				else
+				{
 					Log.Warning(nameof(FileImageSourceHandler), "Could not find image or image file was invalid: {0}", imagesource);
+					return SetImagePlaceholderAsync(imageView, placeholder);
+				}
 			}
 			else
 			{
 				var drawable = ResourceManager.GetDrawable(imageView.Context, file);
 				if (drawable != null)
 					imageView.SetImageDrawable(drawable);
 				else
+				{
 					Log.Warning(nameof(FileImageSourceHandler), "Could not find image or image file was invalid: {0}", imagesource);
+					return SetImagePlaceholderAsync(imageView, placeholder);
+				}
 			}
 
 			return Task.FromResult(true);
 		}
 
+		Task SetImagePlaceholderAsync(ImageView imageView, ImageSource placeholder, CancellationToken cancellation = default(CancellationToken))
+		{
+			string file = ((FileImageSource)placeholder).File;
```

Comment:
What if it's uri image source?

Reply:
For this feature, I'm assuming that images will be in the local resource. I think that no makes sense to get the error and loading PlaceholderImage from an external resources.

Reply:
So i think we should check if it's a FileImageSource if it's not send a proper exception

Reply:
@pictos this way we can't load FontImageSource right ?